### ISSUE TITLE
MKP v-1.7.2 - iPhone 6 (10.3.1) Entry field issues

### ISSUE THREAD

**Issue by kuspie** [MiniKeePass/MiniKeePass#560]
Prior to the last 2 app updates MKP was working properly on my iPhone.   Unfortunately, now at at least 3 of my saved website logins (Allstate Insurance and Home Depot, and Home Depot Pro) when entering the user/pass with the icons it's as if they aren't entered at all because even though I see the entries in the proper fields, when I hit 'login' I get a prompt that says "check user/pass" and the entries are in red as is the prompt to "add user, add pass" for those fields. As if the entries aren't there. At one site as soon as I enter the "user" it automatically goes red and prompts me to enter my user name again, which I just did. It just doesn't see the entries. This started with the update in late September and it has not changed with the latest update in October. FYI, the Windows desktop version of KP works fine.  Within the MKP app  deleted both the offending accounts and added them back. They still produce the same problem. Is this a MKP issue or a website issue? So far, of the dozens of website account logins I have I've found this issue with just the 3 that I named. For these websites I also have their iPhone apps (among others) so I thought there may be a conflict.  I removed both the Allstate and Home Depot apps from the phone.  It did not help.  As stated, I have several other sites in MKP for which I have the individual phone apps as well, several banking sites, other insurance company sites, Lowes, etc, but none of those sites are giving me the problem within MKP that I experience at Allstate and Home Depot.

**Comment by tssmith7** [comment]
I tested the old version of MKP 1.6.3 and it doesn't work with Home Depot Pro either.  I didn't test the others.  The login form somehow checks that the username and password were actually typed from the keyboard or something.  If you change just one letter of the values MKP puts into the fields then it works.  Erase then retype the last letter of both the username and the password and it will work.

**Comment by kuspie** [comment]
Yes that works at the other 2 sites as well.  Thanks for the workaround!
